packages/dx-scheduler-core/src/plugins/common/computeds.ts

```diff
@@ -54,15 +55,15 @@ export const timeScale: TimeScaleFn = (
 };
 
 export const availableViewNames: PureComputed<
```

Comment:
Be careful with rename variables and functions. In this case, you changed the returned value's type and must change the function name also.

Also, you should change it in the tests